tests/phpunit/tests/Compatibility/Toolset/Layouts/TestHooks.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace WPML\PB\Compatibility\Toolset\Layouts;
+
+/**
+ * @group compatibility
+ * @group toolset
+ * @group layouts
+ */
+class TestHooks extends \OTGS\PHPUnit\Tools\TestCase {
+
+	/**
+	 * @test
+	 */
+	public function itImplementActionLoader() {
+		$subject = new Hooks();
+		$this->assertInstanceOf( '\IWPML_Action', $subject );
```

Comment:
$this->assertInstanceOf( \IWPML_Action::class, $subject );
might be useful in case refactoring of name of class